### ISSUE TITLE
Component added to dom twice if parent component does forceUpdate during componentDidMount

### ISSUE THREAD

**Issue by billneff79** [developit/preact-portal#9]
```
class ParentForceUpdateOnMount extends Component {

			componentDidMount() {
				this.forceUpdate();
			}

			render() {
				return (
					<Portal into="body"><div>hello</div></Portal>
				);
			}
		}
```

If you run this code against the current code base, you will end up with:

```
<body>
    <div>hello</div>
    <div>hello</div>
</body>
```

**Comment by billneff79** [comment]
Fixed in PR #8

**Comment by developit** [comment]
Thanks!
